### PR TITLE
Add a docset for the Ghidra scripting API

### PR DESCRIPTION
**Pull request by rgov**

### CODE REVIEW

File: docsets/Ghidra/docset.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Ghidra",
+    "version": "9.0.4",
+    "archive": "Ghidra.tgz",
+    "author": {
+        "name": "Ryan Govostes",
+        "link": "https://github.com/rgov"
+    },
+    "aliases": []
```
Comment (Kapeli) [inline docsets/Ghidra/docset.json]: You're missing a comma at the end of this line.

**Comment by Kapeli** [comment]
If you want an icon you need to include a icon.png and icon@2x.png with sizes 16x16 and 32x32.

**Comment by rgov** [comment]
@Kapeli I was confused by the instructions for an icon:

> Include a `icon.png` and `icon@2x.png` with sizes 16x16 and 32x32 or simply delete the sample icon if you don't want an icon at all.

But this page, https://kapeli.com/docsets#addingicon, says:

> The size of the icon should be 16x16 or 32x32. For Retina display support, you can either use a single 32x32 icon or 2 separate icons: `icon.png` (16x16) and `icon@2x.png` (32x32).

Do you want the icon *both* inside the .docset, *and* inside the folder in this repo? Why can't your processing of this repo just extract the icon from the .docset?

**Comment by Kapeli** [comment]
If you're adding an icon for this repo, you need to follow the instructions provided in this repo.

If you're adding an icon inside a docset, you need to follow the instructions at https://kapeli.com/docsets#addingicon. For the purpose of this repo, the icon inside the docset is not used at all. You can add it or not, it doesn't matter.

> Why can't your processing of this repo just extract the icon from the .docset?

This is not currently supported. I'll consider supporting it if more contributors run into issues with it.

**Pull request closed without merging**